crates/astria-core/src/protocol/fees/v1alpha1.rs

```diff
@@ -10,6 +10,7 @@ use crate::{
 #[error("failed validating on-wire type `{on_wire}` as domain type")]
 pub struct FeeComponentError {
     on_wire: String,
+    #[source]
```

Comment:
nit: you can just rename `inner` to `source` and then you don't need to tag it with `#[source]`